src/core-services/account/util/canAddAccountToGroup.js

```diff
@@ -31,5 +28,7 @@ export default async function canAddAccountToGroup(context, group) {
     !!ownerGroup && contextUserAccount && contextUserAccount.groups.includes(ownerGroup._id)) ||
     userHasPermission("reaction:legacy:shops", "owner", { shopId }); // TODO(pod-auth): update this to figure out what to do with "owner"
 
-  return isOwnerAccount || _.difference(groupPermissions, user.roles[shopId] || []).length === 0;
+  const hasPermissionToAddAccountToGroup = await isOwnerAccount || await userHasPermission("reaction:legacy:groups", "manage:accounts", { shopId });
```

Comment:
The big comment above is outdated. Also, if we can get rid of the "owner" permission as I suggested in another comment, then we won't need the `isOwnerAccount` part here, and this becomes a normal simple permission check.